common/app/views/fragments/audio/containers/flagshipContainer.scala.html

```diff
@@ -57,9 +57,26 @@
                     @getLatestEpisode(allCards).map { card =>
                         <div class="fc-item--type-media">
                             <div class="fc-podcast-container__episode">
-                                <div class="fc-podcast-container__episode-image">
-                                    <img src="@FlagshipFrontContainer.AlbumArtUrl"/>
-                                </div>
+
+                                @if(ActiveExperiments.isParticipating(PodcastImage)){
+                                    @card.displayElement match {
+                                        case Some(InlineImage(image)) => {
+                                            <div class="fc-podcast-container__episode-image-story">
+                                                @itemImage(
+                                                    image,
+                                                    inlineImage = true,
+                                                    widthsByBreakpoint = Some(card.squareImageWidthsByBreakpoint)
+                                                )
+                                            </div>
+                                        }
+                                        case _ => {}
```

Comment:
Just in case, do we want to fallback to the album art URL here?

Reply:
Hmm this will stop the test working. But i suppose stops the front breaking. Which i guess is more important. Will do this